### DISCUSSION TITLE
Landing Page

### DISCUSSION THREAD

**Discussion by ivannaprata** [EasyRoomCompany/Guardian#67] [General]
@christiansousadev como está o andamento da Landing Page? Deu certo trocar a referência de cor? 

Irá ter link para as redes sociais dos desenvolvedores, Data Analytics e Marketing?

**Comment by christiansousadev** [comment]
Pretendo finalizar ainda hoje a Homepage. As cores estão sendo padronizadas de acordo com a logo da empresa.

**Comment by kevinDsousa** [comment]
@christiansousadev Acredito que tu possa colcoar a landpage no repositório da organização, creio que ela nao precise estrar atrelada ao sistema de fato, interage com o @gabrielsousarod ver com ele se as imagens estão aprovadas, precisamos da aprovação do time de marketing.

**Reply by gabrielsousarod** [reply]
cuida @christiansousadev chama no zap! te mandei lá uma landpage com umas imagens de referência.

**Reply by christiansousadev** [reply]
tudo bem
